src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -321,6 +324,30 @@ public void finalCheckInItemHold(UUID transactionId, UUID servicePointId) {
     }
   }
 
+  @Override
+  public InnReachTransactionDTO transferLocalHold(UUID transactionId, UUID itemId) {
+    var transaction = fetchTransactionOfType(transactionId, LOCAL);
+    var hold = transaction.getHold();
+    var item = fetchItemById(itemId);
+
+    transaction.setState(TRANSFER);
+    hold.setFolioItemId(item.getId());
+
+    var requestId = hold.getFolioRequestId();
+    var request = requestService.findRequest(requestId);
+    if (!itemId.equals(request.getItemId())){
+      requestService.moveItemRequest(requestId, item);
```

Comment:
Is the permission needed to perform this request listed in endpoint definition in ModuleDescriptor?